src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/FeatureToggleService.java

```diff
@@ -66,6 +67,10 @@ public boolean isAutomateSendOrderEnabled() {
         return isFeatureEnabled(AUTOMATE_SEND_ORDER);
     }
 
+    public boolean isSendToFRCEnabled() {
+        return isFeatureEnabled(SEND_TO_FRC);
+    }
```

Comment:
Might just be worth adding a comment to this to explain exactly what it's for so people in future don't decide to remove it